src/web/src/components/Posts/PostAvatar.tsx

```diff
@@ -59,7 +59,7 @@ const PostAvatar = ({ name, img, url = '' }: AvatarProps) => {
       .join('');
     return (
       <div>
-        {url.length > 0 ? (
+        {url.length > 0 && url != undefined ? (
```

Comment:
Think just `url.length > 0 && url` is fine enough

Reply:
`url?.length && (...)`